cg/services/order_validation_service/constants.py

```diff
@@ -18,7 +18,7 @@ class ElutionBuffer(StrEnum):
 MINIMUM_VOLUME, MAXIMUM_VOLUME = 20, 130
 
 
-class ExtractionMethod(Enum):
+class ExtractionMethod(StrEnum):
```

Comment:
Changes from Enum to StrEnum were needed because the LIMS parsing was expecting strings.